### DISCUSSION TITLE
feature request - private pages

### DISCUSSION THREAD

**Discussion by will2hew** [docmost/docmost#1624] [Ideas]
We extensively make use of the "private" pages feature of Notion, where a page belongs to a user not to a specific space. 

This would be a good feature from my point of view. Possibly under the hood it's really just a space where only the user is added, but a helpful feature nonetheless.

**Comment by enchove** [comment]
This is a really good idea.
It would be great if admins could set permissions for users to have a personal page, how much space they can use on their personal page, if they can upload files, etc.

**Comment by benjamindahl** [comment]
+1

**Comment by dveb8886** [comment]
How different would this be from the other requests on here for per-page permissions? this would just be a page you don't give anyone permission to access
